### PR TITLE
db: add table for last sync and fix default values

### PR DESCRIPTION
**Pull request by okainov**
In order to switch to OKAPI we need to save last revision.
Previous approach was using timestamps and doesn't work any more.

Also since currently changelog may be partial, we need to remove NOT NULL
constraints from gk-waypointy table, so partial update queries may work

### CODE REVIEW

**Comment by okainov** [comment]
@kumy what is the proper way for proposing DB changes? Should be ok for now, but probably  we should have proper "migrations" folder...

File: docker/mariadb/02_add_waypoint_sync.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  CHANGE COLUMN `typ` `typ` VARCHAR(200) NULL COLLATE 'utf8_polish_ci' AFTER `owner`,
+  CHANGE COLUMN `link` `link` VARCHAR(255) NULL COLLATE 'utf8_polish_ci' AFTER `kraj`;
+
+
+CREATE TABLE `gk-waypointy-sync` (
+                                   `service_id` VARCHAR(5) NOT NULL,
+                                   `last_update` VARCHAR(15) NULL DEFAULT NULL
+)
+  COMMENT='Last synchronization time for GC services'
+  COLLATE='utf8_general_ci'
```
Comment (kumy) [inline docker/mariadb/02_add_waypoint_sync.sql]: please convert to `utf8mb4_unicode_ci`, it'll be the default after #225

**Comment by kumy** [comment]
> @kumy what is the proper way for proposing DB changes? Should be ok for now, but probably we should have proper "migrations" folder...

We do not update the schema a lot, acting like this would be sufficient right now.

**Comment by kumy** [comment]
> Also since currently changelog may be partial, we need to remove NOT NULL
constraints from gk-waypointy table, so partial update queries may work

Does it really hurt for `UPDATE` queries?

**Comment by okainov** [comment]
>Does it really hurt for UPDATE queries?

When UPDATE contains `ON DUPLICATE..` part, then yes. The link I came around when I was developing was https://stackoverflow.com/a/17479690/1657819, so I just made the change and continued.
However now I found yet another suggestion https://stackoverflow.com/a/44000546/1657819 so maybe I can update the queries instead of messing with the schema... Can check later

File: docker/mariadb/02_add_waypoint_sync.sql (hunk 1 above)
Comment (okainov) [reply docker/mariadb/02_add_waypoint_sync.sql]: done

**Pull request merged**